host-bin/enter-chroot

```diff
@@ -684,8 +684,17 @@ if [ -n "$NOLOGIN" ]; then
         fi
     fi
 else
-    # Run rc.local
+    # Check and run rc.local
     if [ -n "$firstrun" -a -x "$CHROOT/etc/rc.local" ]; then
+        ( 
+            sleep 10
+            if [ ! $(ps -ef | grep -c /etc/rc.local) -eq 1 ]; then
+            echo "
+                        
+You rc.local is running for too long...
```

Comment:
A simple `echo "Still running the chroot's /etc/rc.local..." 1>&2` is probably sufficient.